src/context.rs

```diff
@@ -514,7 +519,7 @@ impl Context {
                    AND c.blocked=0
                    AND ct.blocked=0
                    AND (m.txt LIKE ? OR ct.name LIKE ?)
```

Comment:
i just noticed, that has to read `OR ct.name LIKE ? OR ct.authname LIKE ?` (and bind below changed, same some lines above)